src/main/java/com/squad/salesmanager/domain/ProductCategory.java

```diff
@@ -1,4 +1,6 @@
 package com.squad.salesmanager.domain;
 
 public enum ProductCategory {
+    PackedSweets, WeightCandy, Biscuit, Shtuchka, PackagedSweets, coffe
```

Comment:
The enumeration set of objects (choices) should be all uppercase letters
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums/3069863